### ISSUE TITLE
Error: 0xC004F069 On a computer running Microsoft Windows non-core edition

### ISSUE THREAD

**Issue by wojacket** [ekistece/vlmcsd-autokms#6]
According to this [thread](https://social.technet.microsoft.com/Forums/windowsserver/en-US/b96ce945-54e8-4445-bc8f-f0c496912268/kms-error-0xc004f069-on-a-computer-running-microsoft-windows-noncore-edition-run-sluiexe-0x2a?forum=winserver8gen) it's either a time/region error or a volume media with a non-volume key error, and I don't believe it's either. I've tried both the Windows 10 Pro key and Windows 10 LTSC key, it says it works but doesn't.  
![Not an error](https://user-images.githubusercontent.com/53503216/62185286-cc10d200-b32f-11e9-8dbe-63536193711d.png)
Apparently it doesn't think I have the right version of Windows, but I do.

**Comment by NSTAdventure** [comment]
yeah about these error i have no idea to resolve it (IF SOMEONE GET THESE
PROBLEM)

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Pada tanggal Sel, 30 Jul 2019 pukul 22.24 wojacket <notifications@github.com>
menulis:

**Comment by wojacket** [comment]
Does it want me to have a matching .ISO?

**Comment by NSTAdventure** [comment]
Maybe yes?

Pada Rabu, 31 Juli 2019, wojacket <notifications@github.com> menulis:

**Comment by wojacket** [comment]
That or it didn't set up properly for whatever reason

**Comment by NSTAdventure** [comment]
Hffff i hate this -_- i need to be patient first before we can use of again
for now auto KMS still error -_-

Pada Sabtu, 03 Agustus 2019, wojacket <notifications@github.com> menulis:

**Comment by NSTAdventure** [comment]
My buddy i recommended this to you if you looking for activation use this
tutorial for now
https://youtu.be/9oXT6GoqFpI

Pada Sabtu, 17 Agustus 2019, Victor Kurniawan Santoso <kokochromo@gmail.com>
menulis:

**Comment by mchenx** [comment]
@wojacket  you are using wrong edition and product key, try dism /online /get-targeteditions to see edition you can upgrade with and run dism /online /set-edition:xxxxx /productkey:xxxxx to upgrade edition should solve your problem
